packages/lit-html/package.json

```diff
@@ -155,6 +152,127 @@
     "!/development/test/",
     "/directives/"
   ],
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts:types",
+        "build:version-stability-test"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo",
+        "tsconfig.polyfill-support.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "directives/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "async-directive.js{,.map}",
+        "directive-helpers.js{,.map}",
+        "directive.js{,.map}",
+        "experimental-hydrate.js{,.map}",
+        "lit-html.js{,.map}",
+        "polyfill-support.js{,.map}",
+        "private-ssr-support.js{,.map}",
+        "static.js{,.map}",
+        "directives/*.js{,.map}"
+      ]
+    },
+    "build:version-stability-test": {
+      "command": "rollup -c rollup-version-stability-test.config.js",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup-version-stability-test.config.js",
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "version-stability-build"
+      ]
+    },
+    "checksize": {
+      "command": "rollup -c --environment=CHECKSIZE",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": []
+    },
+    "check-version": {
+      "command": "node scripts/check-version-tracker.js",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "scripts/check-version-tracker.js",
+        "package.json",
+        "src/lit-html.ts"
+      ],
+      "output": []
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "command": "cd ../tests && MODE=dev npx wtr \"../lit-html/development/**/*_test.(js|html)\"",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "../tests/web-test-runner.config.js",
+        "../tests/wtr-config.js"
+      ],
+      "output": []
+    },
+    "test:prod": {
+      "command": "cd ../tests && MODE=prod npx wtr \"../lit-html/development/**/*_test.(js|html)\" \"../lit-html/src/test/version-stability_test.js\"",
+      "dependencies": [
+        "build:rollup",
```

Comment:
tbh, the implicit use of dependencies files messes with me a bit when reviewing. `build:rollup` doesn't build the test sources directly, but it depends on `build:ts` which does. I think I'd like to see a direct dependency here on `build:ts` to make that more clear.

Reply:
Hmm, I guess I can see that, but OTOH it's genuinely testing the direct output of `build:rollup`, and not at all testing the direct output of `build:ts` -- so couldn't you make the case that the more minimal style is making that fact a little more apparent?

Reply:
Oh, actually I just re-read your comment and realized you were talking about the *test* sources. That's a good argument, done.